wiki/Tournaments/TMC/4th/en.md

```diff
@@ -177,12 +177,17 @@ The weights for the Qualifiers are as follows:
 | Finals | 13 | 17 | 8 rice, 5 LN, 3 hybrid, 1 tiebreaker |
 | Grand Finals | 13 | 17 | 8 rice, 5 LN, 3 hybrid, 1 tiebreaker |
 
-2. Each round will have its own mappool.
-   - The Group Stage will last for 2 weeks and use the same mappools.
-   - 2 rice maps, 1 LN map, and 1 of the HB maps in the group stage mappool will be made as *classic* picks, meaning they are reused from previous or other tournaments.
-   - 1 rice map and 1 LN map in the other bracket stage mappools will be made as *X* picks, meaning they are more difficult than the other maps in the mappool.
-3. The split double-elimination format will be used in the group stage, meaning the top seed (#1 – #16) and high seed (#17 – #32) will start in the upper bracket, and the mid seed (#33 – #48) and low seed (#49 – #64) will start in the lower bracket.
-   - If fewer than 70 teams play the Qualifiers, the Group Stage will use the standard double-elimination format, meaning all teams will start in the upper bracket.
+2. Each round will have its own mappool, where the match format will be determined by the mappool that each stage uses.
+   - 2 RC maps, 1 LN map, and 1 of the HB maps in the Group Stage mappool will be made as *classic* picks; picks which have been reused from previous or other tournaments.
+   - 1 RC map and 1 LN map in the other bracket stage mappools will be made as *X* picks; picks that are more difficult than usual in the mappool.
+3. The Group Stage format will be determined by the amount of teams who play in the Qualifier stage. Each group will contain a team from the Top Seeds (#1 - #16), High Seeds (#17 - #32), Mid Seeds (#33 - #48) and Low Seeds (#49 - #64), where each group will play a mini double-elimination bracket. A team who wins their double-elimination group bracket will advance into the Playoff bracket.
+   - If 140+ teams play in the Qualifier stage, the format will be *TBD* (a future announcement will be made regarding this circumstance).
+   - If 70 ~ 140 teams play in the Qualifier stage, a split double-elimination format will be used in the group stage, where 64 teams will split into 16 groups. In each group, the Top and High Seeds will start in the Upper Bracket while the Mid and Low Seeds will start in the Lower Bracket.
+     - The Group Stage will last for 2 weeks.
+   - If 32 ~ 70 teams play in the Qualifier stage, the Group Stage will use a standard double-elimination format, where 32 teams split into 8 groups. All teams will start in the Upper Bracket.
+     - In Winner's Semifinals of the Group Stage, the Group Stage mappool will be used.
+     - In Winner's Finals, Loser's Semifinals and Loser's Finals of the Group Stage, the Round of 32 mappool will be used.
+   - If fewer than 32 teams play in the Qualifier stage, the Group Stage will be cancelled and all teams will automatically advance into the Playoff bracket.
 4. The top 2 teams will advance to a standard double-elimination bracket stage, called *Playoffs*.
```

Comment:
Add clarification based on added revisions made to detail the Group Bracket stage
```suggestion
4. The 2 teams (from the Upper and Lower Bracket) that win their Group Stage Finals will advance to the *Playoffs* bracket; a standard double-elimination bracket stage.
```